frontend/src/components/IssueModeSelector/IssueModeSelector.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Text } from '@highlight-run/ui/components'
+import clsx from 'clsx'
+
+import * as styles from './IssueModeSelector.css'
+
+export const TagSwitch = <T extends string | number>({
```

Comment:
I'm interested in working on #8105. 
Do you have any suggestions regarding how
best to extract this component?